packages/deployments/contracts/contracts/messaging/connectors/admin/AdminHubConnector.sol

```diff
@@ -37,9 +41,18 @@ contract AdminHubConnector is ProposedOwnable, HubConnector {
 
   // ============ Private fns ============
 
-  // function included to match interface but is not used internally, therefore is a no op.
-  function _verifySender(address _expected) internal view override returns (bool) {}
+  /**
+   * @notice Should not be used as this contract has no crosschain counterpart.
+   * @dev Reverts if `Connector.verifySender` is called
+   */
+  function _verifySender(address) internal pure override returns (bool) {
+    require(false, "!supported");
+  }
 
-  // function included to match interface but is not used internally, therefore is a no op.
+  /**
+   * @notice This function is a no-op as this connector does NOT send messages across
+   * chains.
+   * @dev Is a no-op over a revert so `RootManager.sendMessage` does not revert.
+   */
```

Comment:
I was going to say we should make this revert because the try-catch will log it as an error which is maybe expected. But we are changing this system so it's better to make it a no-op.